geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTableV5.scala

```diff
@@ -35,7 +35,7 @@ object AttributeTableV5 extends GeoMesaTable with LazyLogging {
   override def supports(sft: SimpleFeatureType) =
     sft.getSchemaVersion < 6 && SimpleFeatureTypes.getSecondaryIndexedAttributes(sft).nonEmpty
 
-  override val suffix: String = "attr_idx"
+  override val suffix: String = "attr"
```

Comment:
Can you create a new V5 table with GeoMesa 1.2.x?

If not, I'd suggest leaving this along and commenting it.